web/src/pm_list.js

```diff
@@ -104,6 +104,9 @@ export function expand() {
     $("#toggle_private_messages_section_icon").addClass("fa-caret-down");
     $("#toggle_private_messages_section_icon").removeClass("fa-caret-right");
     update_private_messages();
+    if (topic_zoom.is_zoomed_in()) {
+        topic_zoom.zoom_out();
+    }
```

Comment:
I think we want to do this first -- in the case that `update_private_messages` is doing anything fancy with scroll position, it'll be much easier if this is already closed.